chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -70,11 +70,13 @@
     block do
       require "pathname"
       path = Pathname.new(dump_location).split.first.to_s
-      available = `df #{path} | tail -n 1 | sed "s/ \\+/ /g" | cut -d " " -f 4`
+      available = `df -k #{path} | tail -n 1 | sed "s/ \\+/ /g" | cut -d " " -f 4`
       db_size = `su - postgres -c 'pg_dumpall | wc -c'`
+      # Use kilobytes and add some reserve (10 MB)
+      db_size = db_size.to_i / 1024 + 10240
       if db_size.to_i > available.to_i
         message = "Not enough space for the Database dump on node #{node.name}!\n" \
-          "Database size is: #{db_size}B, available space on #{path} is only #{available}B."
+          "Database size is: #{db_size}kB, available space on #{path} is only #{available.to_i}kB."
```

Comment:
I am reasonably sure that is using binary and not decimal units, so is `KB`, `KiB` or `K`